Sprint-1/exercises/paths.js

```diff
@@ -16,3 +16,6 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+
+var dir = filePath.slice(0, lastSlashIndex);
+var ext = base.slice(4);
```

Comment:
How would you change your code so that it can also work for file paths like
`/path1/.git/filename.js`
`/path1/path2/package.json`
?

Reply:
I have updated its dynamic now.
1-  /path1/.git/filename.js
2- /path1/path2/package.json
the expected output :
The base part files1 and 2 paths are   filename.js , package.json
The dir part of file 1 and 2 paths are   /path1/.git,   /path1/path2
The ext part of file 1 and 2 paths are  .js,  .json